src/Standards/Squiz/Tests/PHP/EmbeddedPhpUnitTest.php

```diff
@@ -190,6 +191,17 @@ public function getErrorList($testFile='')
                 22 => 2,
             ];
 
+        case 'EmbeddedPhpUnitTest.24.inc':
+            $shortOpenTagDirective = (bool) ini_get('short_open_tag');
+            if ($shortOpenTagDirective === true) {
+                return [
+                    18 => 1,
+                    20 => 1,
+                ];
+            } else {
+                return [];
+            }
```

Comment:
```suggestion

            return [];
```

There is no need for an `else` here.